pkg/virt-controller/watch/vmi.go

```diff
@@ -394,6 +402,8 @@ func (c *VMIController) hasOwnerVM(vmi *virtv1.VirtualMachineInstance) bool {
 }
 
 func (c *VMIController) updateStatus(vmi *virtv1.VirtualMachineInstance, pod *k8sv1.Pod, dataVolumes []*cdiv1.DataVolume, syncErr syncError) error {
+	key := fmt.Sprintf("%s/%s", vmi.Namespace, vmi.Name)
+	defer virtControllerVMIWorkQueueTracer.StepTrace(key, "updateStatus", trace.Field{Key: "VMI Name", Value: vmi.Name})
```

Comment:
just double checking, did you want this line to execute after update status completes, or at the beginning of when it starts?

Reply:
The trace library says to call it after.  We don't have to, but I think this pattern - `defer ...` at the top - will avoid accidents.
```
// Step adds a new step with a specific message. Call this at the end of an execution step to record
// how long it took.
```

Reply:
Having the `step` there is still valuable because it labels the step were doing - tracing the time between the end of `sync()` and the end of `updateStatus()`.  Otherwise, the time between `sync()` and `stop trace` has no label when it's running `updateStatus()`. Also, someone could add code in the future that runs after `updateStatus()` and it could get its own `Step` making the time after `updateStatus()` useful.